examples/jn516x/tsch/common-conf.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2014, Swedish Institute of Computer Science.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. Neither the name of the Institute nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE INSTITUTE AND CONTRIBUTORS ``AS IS'' AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED.  IN NO EVENT SHALL THE INSTITUTE OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+ * OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ *
+ */
+/**
+ * \author Simon Duquennoy <simonduq@sics.se>
+ */
+
+#ifndef __COMMON_CONF_H__
+#define __COMMON_CONF_H__
+
+/* Global config flags */
+
+#define WITH_TSCH 1
+#define WITH_TSCH_SECURITY 0
+#define TSCH_LOG_CONF_LEVEL 2
+#define WITH_COAP_RESOURCES 0
+
+#undef ENABLE_COOJA_DEBUG
+#define ENABLE_COOJA_DEBUG 0
+
+#if WITH_COAP_RESOURCES
+#define UIP_DS6_WITH_LINK_METRICS 1
+#define PLEXI_CONFIG_RPL 1
+#if WITH_TSCH
+#define PLEXI_CONFIG_TSCH            2
+#endif /* WITH_TSCH */
+#define RICH_INTERFACE PLEXI_CONFIG_RPL | PLEXI_CONFIG_TSCH
```

Comment:
above are a few plexi-related defines, should be removed as plexi is not in upstream contiki